content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,313 @@
+
+| Backend property             | Note |
+|------------------------------|------|
+| `etcd-trusted-ca-file`       | Self explanatory. |
+| `etcd-cert-file`             | Self explanatory. |
+| `etcd-key-file`              | Self explanatory. |
+| `etcd-client-cert-auth`      | Strongly recommended to use `true` setting |
+| `etcd-advertise-client-urls` | list of https URLs to to advertise for Etcd Replicators, accessible by other backends in the federation e.g. `https://sensu.beta.example.com:2378` |
+| `etcd-listen-client-urls`    | list of https URLs to listen on for Etcd Replicators, e.g. `https://0.0.0.0:2379` to listen on port 2739 across all ipv4 interfaces |
+
+_NOTE: You *must* provide non-default values for `etcd-advertise-client-urls` and `etcd-listen-client-urls` backend parameters, as the default values are not suitable for use under federation._
+
+### Step 2 Configure shared token signing keys
+
+Whether federated or stand-alone, Sensu backends issue JSON web tokens (JWTs) to users upon successful authentication. These tokens include a payload which describes the username and group affiliations, used to determine permissions based on configured [RBAC policy][10].
+
+In a federation of Sensu backends, each backend needs to have the same public/private key pair. These asymmetric keys are used to crypotgraphically vouch for the user's identity in the JWT payload. This use of shared JWT keys enables clusters to grant users access to Sensu resources according to their local policies but without requiring User resources to be present uniformly across all clusters in the federation.
+
+By default a Sensu backend automatically generates an asymmetric key pair for signing JWTs and stores it in the etcd database. When configuring federation, you need to generate keys as files on disk so they can be copied to all backends in the federation.
+
+Use use the `openssl` command line tool to generate a P-256 Elliptic Curve private key:
+
+```shell
+openssl ecparam -genkey -name prime256v1 -noout -out jwt_private.pem
+```
+
+Then generate a public key from the private key:
+
+```shell
+openssl ec -in jwt_private.pem -pubout -out jwt_public.pem
+```
+
+For this guide, we'll put JWT keys into `/etc/sensu/certs` and use the [`jwt-private-key-file` and `jwt-public-key-file` attributes][4] in `/etc/sensu/backend.yml` to specify the paths to these JWT keys:
+
+{{< highlight yml >}}
+jwt-private-key-file: /etc/sensu/certs/jwt_private.pem
+jwt-public-key-file: /etc/sensu/certs/jwt_public.pem
+{{< /highlight >}}
+
+After updating the backend configuration in each cluster, restart `sensu-backend` for those settings to take effect.
+
+### Step 3 Add a cluster role binding and user
+
+To prove out our configuration, we'll provision a User and a ClusterRoleBinding in the `gateway` cluster.
+
+First, confirm that sensuctl is configured to communicate with the `gateway` cluster using `sensuctl config view` to see the active configuration.
```

Comment:
I think it would be good to include an example that shows what the response should look like if sensuctl is configured correctly here.

{{< highlight shell >}}
sensuctl config view

{this would be the example that shows the response if correctly configured}
{{< /highlight >}}